extensions/skins/conversation_v1/static/conversation.css

```diff
@@ -308,6 +311,10 @@ md-card.conversation-skin-supplemental-card {
   margin-top: 2px;
   width: 46px;
 }
+.conversation-skin-responses-animate-slide{
+  padding-right: 56px;
+  width: 528px;
```

Comment:
Why do you need to specify the width here? It looks odd on larger screens (try the Welcome exploration, second card).

![large-width](https://cloud.githubusercontent.com/assets/10575562/10239266/e7721ba2-687f-11e5-812d-5a36abe97518.png)